lib/addons/amp/collector-config-amp.yaml

```diff
@@ -0,0 +1,358 @@
+#
+# OpenTelemetry Collector configuration
+# Metrics pipeline with Prometheus Receiver and AWS Remote Write Exporter sending metrics to Amazon Managed Prometheus
+#
+
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: my-collector-amp
```

Comment:
Sounds like a hello world name for this. Can we let customer pass the name or use something more professional?

Reply:
This is directly from the collector deployment process `curl -o collector-config-amp.yaml https://raw.githubusercontent.com/aws-observability/aws-otel-community/master/sample-configs/operator/collector-config-amp.yaml` which is on the page https://docs.aws.amazon.com/eks/latest/userguide/deploy-deployment.html we want to stay consistent with what we have in the deployment process.

Reply:
I fixed it to make it professional.